data/json/vehicles/vans_busses.json

```diff
@@ -152,6 +152,84 @@
+      { "x": 2, "y": 0, "parts": [ "frame_horizontal_front", "halfboard_cover_left" ] },
+      { "x": 2, "y": 1, "parts": [ "frame_cover", "halfboard_cover", "engine_v6" ] },
+      { "x": 2, "y": 1, "parts": [ "alternator_truck", "battery_car" ] },
+      { "x": 2, "y": 2, "parts": [ "frame_horizontal_front", "halfboard_cover_right" ] },
+      { "x": 2, "y": 3, "parts": [ "frame_ne", "halfboard_ne", "headlight" ] },
+      { "x": -1, "y": -1, "parts": [ "frame_sw", "board_nw" ] },
+      { "x": -1, "y": -1, "part": "tank", "fuel": "gasoline" },
+      { "x": -1, "y": 0, "parts": [ "frame_cross", "board_horizontal_front" ] },
+      { "x": -1, "y": 1, "parts": [ "frame_cross", "storage_battery", "board_horizontal_front" ] },
+      { "x": -1, "y": 2, "parts": [ "frame_cross", "board_horizontal_front" ] },
+      { "x": -1, "y": 3, "parts": [ "frame_se", "board_ne" ] },
+      { "x": -1, "y": 3, "part": "tank", "fuel": "gasoline" },
+      { "x": -2, "y": -1, "parts": [ "frame_vertical_left", "board_vertical_left", "roof" ] },
+      { "x": -2, "y": 0, "parts": [ "frame_cross", "trunk", "roof" ] },
+      { "x": -2, "y": 1, "parts": [ "frame_cross", "aisle_vertical", "mountable_cooler", "roof" ] },
+      { "x": -2, "y": 2, "parts": [ "frame_cross", "trunk", "roof" ] },
+      { "x": -2, "y": 3, "parts": [ "frame_vertical_right", "board_vertical_right", "roof" ] },
+      { "x": -3, "y": -1, "parts": [ "frame_vertical_left", "board_vertical_left", "roof" ] },
+      { "x": -3, "y": 0, "parts": [ "frame_cross", "trunk", "roof" ] },
+      { "x": -3, "y": 1, "parts": [ "frame_cross", "aisle_vertical", "roof" ] },
+      { "x": -3, "y": 2, "parts": [ "frame_cross", "trunk", "roof" ] },
+      { "x": -3, "y": 3, "parts": [ "frame_vertical_right", "board_vertical_right", "roof" ] },
+      { "x": -4, "y": -1, "parts": [ "frame_vertical_left", "board_wheel_left" ] },
+      { "x": -4, "y": -1, "parts": [ "wheel_mount_medium", "wheel_wide", "roof" ] },
+      { "x": -4, "y": 0, "parts": [ "frame_cross", "trunk", "roof" ] },
+      { "x": -4, "y": 1, "parts": [ "frame_cross", "aisle_vertical", "roof" ] },
+      { "x": -4, "y": 2, "parts": [ "frame_cross", "trunk", "roof" ] },
+      { "x": -4, "y": 3, "parts": [ "frame_vertical_right", "board_wheel_right" ] },
+      { "x": -4, "y": 3, "parts": [ "wheel_mount_medium", "wheel_wide", "roof" ] },
+      { "x": -5, "y": -1, "parts": [ "frame_sw", "board_sw", "roof" ] },
+      { "x": -5, "y": 0, "parts": [ "frame_horizontal_rear", "board_horizontal_rear", "roof" ] },
+      { "x": -5, "y": 1, "parts": [ "frame_horizontal_rear", "muffler", "door_shutter", "beeper", "roof" ] },
+      { "x": -5, "y": 2, "parts": [ "frame_horizontal_rear", "muffler", "board_horizontal_rear", "roof" ] },
+      { "x": -5, "y": 3, "parts": [ "frame_se", "board_se", "roof" ] }
+    ],
+    "items": [
+      { "x": -2, "y": 0, "chance": 25, "item_groups": [ "irradiated_food" ] },
```

Comment:
Probably wouldn't be irradiated food in the refrigerated van.

Reply:
Yeah, produce is generally transported in both refrigerated and non refrigerated trucks, just wanted something to put in it.

I went with irradiated so the food in it would be less likely to be rotten by the time the player finds it.  

What other type would be passable as a relatively common food cargo and have a higher chance of staying fresh ?(seeing as these trucks are designed to keep foods fresh)

Reply:
Potatoes and pumpkins tend to last longer, could also have stuff that's sealed by default (yoghurt, pudding, drinks).
Worked in a supermarket and pretty much anything that wasn't dry store (sometimes even then) or not intended to be refrigerated came in refrigerated trucks.